### DISCUSSION TITLE
Where does the Plug-in store/cache the cell contents?

### DISCUSSION THREAD

**Discussion by emign** [RafaelGB/obsidian-db-folder#532] [Q&A] [answered]
Hi,
I sync my content via a git repo. When I setup a db folder (with data view source) everything shows up fine on my main device. But the correctly synced files only produce the tables on my second device. The contents of the cells are completely missing. 

Thanks for help

**Comment by RafaelGB** [comment, marked as answer]
the plugin does not use cache as such. A query is performed every time the table is opened to dataview. It could be that when you synchronize dataview it has not yet updated said content.

We recommend reloading the app in these cases to force local dataview loading
